front/components/assistant/TagsManager.tsx

```diff
@@ -111,16 +140,30 @@ export function TagsManager({ open, setOpen, owner }: TagsManagerProps) {
           </SheetHeader>
 
           <SheetContainer>
-            <div className="flex w-full flex-row justify-end">
-              <NewTagButton owner={owner} />
-            </div>
-
             {isTagsLoading && (
               <div className="flex flex-row items-center">
                 <Spinner />
               </div>
             )}
-            <DataTable data={rows} columns={columns} />
+            {rows.length > 0 && !isTagsLoading && (
+              <>
+                <div className="flex w-full flex-row justify-end">
+                  <NewTagButton owner={owner} />
+                </div>
+                <DataTable data={rows} columns={columns} />
+              </>
+            )}
+            {rows.length === 0 && !isTagsLoading && (
+              <EmptyCTA
+                action={
+                  <div className="flex flex-row gap-2">
+                    <SuggestTagsButton owner={owner} />
+                    <NewTagButton owner={owner} empty />
+                  </div>
+                }
+                message="No tags have been created yet. Let AI suggest tags for your agents, or add manually."
```

Comment:
NIT: Would be great to repeat the purpose of tags here, feel free to discard: 

"Start organizing agents with tags. Get some suggestions or create them manually."